app/artist/[mbId].tsx

```diff
@@ -23,7 +26,7 @@ const ArtistPage = () => {
     <>
       <Stack.Screen
         options={{
-          title: artistData ? `${artistData?.name}'s latest setlists` : "",
+          title: artistData ? t(i18n)`${artistData?.name}'s latest setlists` : "",
```

Comment:
it's possible we don't need the useLingui hook and this i18n passthru anymore, double check